content/developer/composer/blueprint-creation.md

```diff
@@ -73,6 +73,17 @@ Depending on the property type this can be changed to one of other options:
 
 - **Input** - allows using input value as property value (uses `get_input` intrinsic function)
 - **Secret** - allows using secret value as property value (uses `get_secret` intrinsic function)
+- **Property** - allows using another's node template property as value (uses `get_property` intrinsic function)
+- **System** - allows using tenant name or certain deployment attributes as property value
+- **Environment capability** - allows using existing environment blueprint capability as property value (uses `get_environment_capability` intrinsic function).
+  When this option is active the value is specified with two dropdowns.
+  The first one selects existing environment blueprint existing on the {{< param cfy_manager_name >}}.
```

Comment:
Maybe we could rephrase it a bit so that we don't have "existing" word used twice in this sentence?

Reply:
Done